tool/plugin/lib/plugin.dart

```diff
@@ -883,12 +884,19 @@ class BuildSpec {
   String _parseChangelog() {
     var text = new File('CHANGELOG.md').readAsStringSync();
     var html = markdownToHtml(text);
+
+    // Translate our markdown based changelog into html; remove unwanted
+    // paragraph tags.
     return html
         .replaceAll('</h2><ul>', '</h2>\n<ul>')
-        .replaceAll('<ul><li>', '<ul>\n  <li>')
-        .replaceAll('</li><li>', '</li>\n  <li>')
+        .replaceAll('<ul>\n<li>', '<ul>\n  <li>')
+        .replaceAll('</li>\n<li>', '</li>\n  <li>')
         .replaceAll('</li></ul>', '</li>\n</ul>')
-        .replaceAll('<li>\n<p>', '<li><p>');
+        //.replaceAll('<li>\n<p>', '<li><p>')
```

Comment:
does this comment still need to be here?

Reply:
oops, no

Reply:
done!